### DISCUSSION TITLE
Possibility to add English documentation?

### DISCUSSION THREAD

**Discussion by jiahao-c** [Tencent/wujie#767] [General]
Hello. I work at a company where English is the primary working language. My team is interested in using wujie, but we won't be able to without English documentation. Would it be possible for wujie team to add an English version documentation?

**Comment by yiludege** [comment]
thankou for you interesting，wujie don't provide a  English doc first because my poor English 🥲 ,then maybe there no strong need for micro-front in overseas areas becase i can't find other micro-front frame expect single-spa, last i think wujie proposal still can't resolve all  the micro-front problem and not a perfect proposal

but the soure of wujie is not complex and easy to read，module can easy distinguish by filename，if need help just take it to issue or discussions，but English doc may not add in the short term，other micro-front frame can see [qiankun](https://qiankun.umijs.org/)

**Reply by jiahao-c** [reply]
Thank you for the quick reply, I really appreciate it!
